lib/pltr/v2/reducers/cards.js

```diff
@@ -75,8 +75,24 @@ const cards =
           }),
         ]
 
-      case ADD_LINES_FROM_TEMPLATE:
-        return [...action.cards]
+      case ADD_LINES_FROM_TEMPLATE: {
+        const newCards = action.templateData.cards.map((c) => {
+          // give it the correct beatId
+          if (action.ignoreBeats) {
+            c.beatId = action.cardToBeatIdMap[c.id]
+          } else {
+            // TODO:
+            // think through this some more
+            // c.beatId = action.nextBeatId + c.beatId
+          }
+          c.lineId = action.nextLineId + c.lineId // give it the correct lineId
+          c.id = c.id + action.nextCardId // give it a new id
+          c.fromTemplateId = action.templateData.id
+          return c
```

Comment:
There might be a subtle bug here.
This modifies the cards on the template.
If you add the same template to two separate timelines, then I think you'll end up modifying the cards on both timelines(!)
Probably best to clone the card to be safe.